test/test_queries_multirow.txt

```diff
@@ -0,0 +1,4 @@
+select to_string(to_timestamp('2009-09-17T17:56:06.234567Z'), substring(' athmywopgss-nghjkl', 3, 10)) from stdin;
```

Comment:
in multi-row tests , its not correct to use fixed value, should use a column reference